grpc-gateway/service/service.go

```
@@ -47,13 +47,13 @@ func New(ctx context.Context, config Config, fileWatcher *fsnotify.Watcher, logg
 
 	pool, err := ants.NewPool(config.Clients.Eventbus.GoPoolSize)
 	if err != nil {
-		server.Close()
+		_ = server.Close()
```

This could be problematic if the implementation ever changes and returns a real error. I'm 99% sure that the implementer won't think about needing to update all calls of `server.Close` to check error.

---

fixed

---

There are `s.Close()` and `grpcServer.Close()` calls that ignore the error. :innocent:

---

ok fixed :)

---

Look good. ;)